### PR TITLE
Added globalize gem

### PR DESCRIPTION
**Pull request by drewpotter**
Opening up a PR so that @spencerldixon can see the branch as it progresses and add comments.

### CODE REVIEW

File: test/controllers/classifications_controller_test.rb [hunk 1]
```diff
@@ -27,7 +27,7 @@ class ClassificationsControllerTest < ActionDispatch::IntegrationTest
       post classifications_url, params: { classification: { description: @classification.description, name: @classification.name, results_description: @classification.results_description } }
     end
 
-    assert_redirected_to classification_url(Classification.last)
+    assert_redirected_to classification_url(Classification.last, locale: :en)
```
Comment (spencerldixon) [inline test/controllers/classifications_controller_test.rb]: This doesn't look good, is this really necessary?

File: app/controllers/responses_controller.rb [hunk 2]
```diff
@@ -19,6 +19,7 @@ def create
     @response            = Response.new(response_params)
     @response.survey     = @survey
     @response.ip_address = request.remote_ip
+    @response.language   = params[:locale] if I18n.available_locales.map(&:to_s).include?(params[:locale])
```
Comment (spencerldixon) [inline app/controllers/responses_controller.rb]: Do we need the if bit?

Comment (spencerldixon) [reply app/controllers/responses_controller.rb]: What if locale doesn't exist?

Comment (drewpotter) [reply app/controllers/responses_controller.rb]: Many thanks @spencerldixon I can rewrite this line.

**Pull request merged**